### DISCUSSION TITLE
extra Dialog box reduces time to start blocking websites

### DISCUSSION THREAD

**Discussion by dnecklesportfolio** [SelfControlApp/selfcontrol#864] [General]
Is it possible to not have to see this dialog box everytime.
<img width="263" alt="image" src="https://github.com/SelfControlApp/selfcontrol/assets/2013689/915a0a62-203e-4933-9b76-6f5357e737d7">

**Comment by slambert** [comment]
There's [an issue for this here](https://github.com/SelfControlApp/selfcontrol/issues/500)

**Reply by dnecklesportfolio** [reply]
I didn't use the right search term, thanks a lot.
